### PR TITLE
Kernel Implementation

### PR DESCRIPTION
**Pull request by maxwellE3**
Kernel Implementations

### CODE REVIEW

File: doc/05-component-kernel-implementation/05-component-kernel-implementation.md [hunk 1]
```diff
@@ -111,9 +109,6 @@ do good work.
 > representation now and justify it. Feel free to also use this space to
 > discuss how that representation will be restricted (i.e., by convention)
 > and interpreted (i.e., by correspondence).
```
Comment (jrg94) [inline doc/05-component-kernel-implementation/05-component-kernel-implementation.md:111]: Make sure you argue why you selected the representation you did.

**Review by jrg94 — APPROVED** [review]
Excellent work! Changes look good too.

**Comment by jrg94** [comment]
Just missing the justification for the choice of Queue.

File: doc/05-component-kernel-implementation/05-component-kernel-implementation.md (hunk 1 above)
Comment (maxwellE3) [reply doc/05-component-kernel-implementation/05-component-kernel-implementation.md:111]: I have added a justification in Chord1L above the instantiation of my rep.

**Review by jrg94 — APPROVED** [review]

**Pull request merged**